MedicalDocManagment.WebUI/app/views/users-search-form.component.html

```diff
@@ -0,0 +1,32 @@
+﻿<h3>Пошук користувачів</h3>
+<form class="form-horizontal">
+    <div class="form-group">
+        <label class="control-label col-sm-2" for="username">Ім'я</label>
+        <div class="col-sm-10">
+            <input type="text" class="form-control" id="username">
+        </div>
+    </div>
+    <div class="form-group">
+        <label class="control-label col-sm-2" for="email">Email</label>
+        <div class="col-sm-10">
+            <input type="email" class="form-control" id="email">
```

Comment:
Please consider using radiobuttons to choose between search possibilities, I believe only one input field should be available if we are going to search only by one parameter at a time.